src/triggers/default_triggers.py

```diff
@@ -8,6 +8,18 @@
 
 _default_triggers = frozendict(
     {
+        "production": Trigger(
+            condition=("service_base_url", consts.RemoteEnvironment.PRODUCTION),
+            worker_disk=128849018880
```

Comment:
```suggestion
            worker_disk=120 * 2 ** 30  # 120 GB
```
or any other option that will make the size in use more clear

Reply:
And please keep that value as const